chacha20_poly1305/Cargo.toml

```diff
@@ -22,3 +22,6 @@ hex = { package = "hex-conservative", version = "0.3.0", default-features = fals
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
+
+[lints.rust]
+unexpected_cfgs = { level = "deny", check-cfg = ['cfg(bench)'] }
```

Comment:
I think we should have it as warn but deny it from CLI, if possible, like other warnings. Anyway IIRC this is used everywhere, so It's a separate topic.

Reply:
Lol, were you around [for the debacle around this lint being introduced?](https://github.com/rust-lang/rust/issues/124800). It's still too soon for me to think about this again.

(I'm partly kidding -- if you were to figure out how to make this warn-only but error in CI, [despite the FUD on the rust blog itself about the configurability of this lint](https://github.com/rust-lang/blog.rust-lang.org/pull/1313), I'd be happy to ACK that PR and copy your cargo invocation into my local CI. But I have zero interest in figuring it out myself.)

Reply:
> Lol, were you around [for the debacle around this lint being introduced?](https://github.com/rust-lang/rust/issues/124800). It's still too soon for me to think about this again.

Yes, it was quite weird that it took that much convincing for them to accept it in `Cargo.toml`. But hey, at least they did.